tmt/steps/discover/fmf.py

```diff
@@ -89,20 +90,30 @@ def wake(self):
     def go(self):
         """ Discover available tests """
         super(DiscoverFmf, self).go()
-        testdir = os.path.join(self.workdir, 'tests')
 
-        # Clone provided git repository
+        # Check url and path, prepare test directory
```

Comment:
Where do we do the `check`? `get` function does it?

Reply:
Well, yeah. The comment is a bit talkative here ;-)